### ISSUE TITLE
Multi front-end Odoo server | ERROR: could not serialize access due to concurrent update

### ISSUE THREAD

**Issue by Elbagoury** [odoo/odoo#43382]
**Impacted versions:**
11,12,13 Community and Enterprise

**Steps to reproduce:**
for high availability and load balancing, I created two Odoo frontend server connecting to one database server and the problem is they run the same cron job twice and causing this issue:
`ERROR: could not serialize access due to concurrent update`

**Comment by gdgellatly** [comment]
Yes, you need to manage this, either by turning cron off on both and creating a third cron only server, or by a few lines of code implementing master/slave. Generally you just have a new cron server spin up when it detects the first failed. Missing cron for a couple of minutes is no big deal for HA. You can actually reproduce this on a single server if you have enough workers and processors.

**Comment by Elbagoury** [comment]
@gdgellatly  @pedrobaeza is there a way to disable the corn job in odoo conf files..? 
I made the max_cron_threads = 0 and still run the corn jobs strange .!!
